managers.py

```diff
@@ -0,0 +1,47 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "authors"
+
+    def create(self, authors: str):
```

Comment:
The `create` method should accept `first_name` and `last_name` parameters instead of a single `authors` parameter. Update the method signature and SQL statement accordingly.